### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JimmyWhitaker**
Added missing steps for deploying the IDE.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +19,7 @@ If you need to customize your IDE deployment more than what `pachctl deploy ide`
 Once deployed, navigate to your instance:
 
 - By default, it should be reachable on port 80 of your cluster's hostname.
-- On minikube, navigate to one of the URLs printed out when you run `minikube service proxy-public --url`.
+- On minikube, navigate to one of the URLs printed out when you run `minikube service proxy-public --url | head -n 1`.
```
Comment (ysimonson) [inline README.md:22]: I discovered recently that this is actually not right: for some reason in minikube, while it prints two URLs, only one of them will work - and it's not necessarily the first, sometimes it's the second.

Comment (JimmyWhitaker) [reply README.md:22]: Is this fixed in #62 ? If so, I'll close this PR.

Comment (ysimonson) [reply README.md:22]: it's not unfortunately, I just wrote a helper script to address this issue in tests

Comment (JimmyWhitaker) [reply README.md:22]: Okay, I'll remove this line and merge the other changes, if you're okay with that.

**Comment by ysimonson** [comment]
This looks good! Just a heads up that the syntax for `pachctl enterprise activate` will change with 1.12 though. I'll file a ticket for that.

**Comment by ysimonson** [comment]
https://github.com/pachyderm/pachyderm-ide/issues/64

**Pull request merged**